app/service/food_service.rb

```diff
@@ -0,0 +1,13 @@
+class FoodService
+
+  def self.conn
+    Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/") do |faraday|
```

Comment:
I think for the connection URL you would end it at `.gov` and then everything after that would go in the `conn.get` method.
Here is where you could also add another param to set page limit to 10. 
```
faraday.params[:pageSize] = 10
```